cogs/economy.py

```diff
@@ -407,31 +407,31 @@ def already_in_list(self, accounts, user):
         display_reels += ">" + reels[0][1] + " " + reels[1][1] + " " + reels[2][1] + "\n"
         display_reels += "  " + reels[0][2] + " " + reels[1][2] + " " + reels[2][2] + "\n"
 
-        slotMsg = ""
+        slotMsg = "\n"
```

Comment:
Don't know if the blame is on Discord, but starting newlines will get stripped now days.
Wrapping some random markup around it seems to allow you to start with newlines again, for now.
~~\n~~